autopush/tests/test_integration.py

```diff
@@ -902,6 +911,45 @@ def test_multiple_delivery_with_single_ack(self):
         yield self.shut_down(client)
 
     @inlineCallbacks
+    def test_multiple_delivery_with_single_ack(self):
+        data = str(uuid.uuid4())
+        data2 = str(uuid.uuid4())
+        client = yield self.quick_register(use_webpush=True)
+        yield client.disconnect()
+        ok_(client.channels)
+        yield client.send_notification(data=data)
+        yield client.send_notification(data=data2)
+        yield client.connect()
+        yield client.hello()
+        result = yield client.get_notification()
+        ok_(result != {})
```

Comment:
I think all the ok_(result != {}) in this file should be ok_(result is not None) like you are doing in the other new tests